xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/DeletedDocuments.xml

```diff
@@ -341,6 +342,46 @@ XWiki.index.trash.documents.confirmDelete = function(event, table, rowIdx) {
       );
   }
 }
+
+/**
+ * Event listener, called when clicking the "delete all" button.
+ *
+ * @param event the link activation event
+ */
+XWiki.index.trash.documents.confirmDeleteAll = function(event) {
+  event.stop();
```

Comment:
We're not supposed to use Prototype.js anymore. In this case it is quite easy to write the event listener and to register the event using Require.js and jQuery, which is the recommended approach.

Reply:
I'll change that in another PR.